.github/workflows/build.yml

```diff
@@ -25,6 +25,21 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Install system packages
+        run: |
+          sudo apt-get update -y
```

Comment:
If you put the "Install WineHQ" step before this one, you could remove this `sudo apt-get update -y` line which could save some build time.

Reply:
Well, then it would not be "Install winehq" any longer :D

The idea was that the brain would read it better top-down, base deps + wine deps.  It also allows the steps to be used independent of each other.  This readability is a trade-off in this case.  I'll move it, but having a measurable "time" would help justify.